x/nodes/types/params.go

```diff
@@ -11,65 +11,77 @@ import (
 // POS params default values
 const (
 	// DefaultParamspace for params keeper
-	DefaultRelaysToTokensMultiplier int64 = 1000
-	DefaultParamspace                     = ModuleName
-	DefaultUnstakingTime                  = time.Hour * 24 * 7 * 3
-	DefaultMaxValidators            int64 = 5000
-	DefaultMinStake                 int64 = 1000000
-	DefaultMaxEvidenceAge                 = 60 * 2 * time.Second
-	DefaultSignedBlocksWindow             = int64(100)
-	DefaultDowntimeJailDuration           = 60 * 10 * time.Second
-	DefaultSessionBlocktime               = 25
-	DefaultProposerAllocation             = 1
-	DefaultDAOAllocation                  = 10
-	DefaultMaxChains                      = 15
-	DefaultMaxJailedBlocks                = 1000
+	DefaultRelaysToTokensMultiplier     int64 = 1000
+	DefaultParamspace                         = ModuleName
+	DefaultUnstakingTime                      = time.Hour * 24 * 7 * 3
+	DefaultMaxValidators                int64 = 5000
+	DefaultMinStake                     int64 = 1000000
+	DefaultMaxEvidenceAge                     = 60 * 2 * time.Second
+	DefaultSignedBlocksWindow                 = int64(100)
+	DefaultDowntimeJailDuration               = 60 * 10 * time.Second
+	DefaultSessionBlocktime                   = 25
+	DefaultProposerAllocation                 = 1
+	DefaultDAOAllocation                      = 10
+	DefaultMaxChains                          = 15
+	DefaultMaxJailedBlocks                    = 1000
+	DefaultServicerStakeFloorMultiplier int64 = 15000000000
+	DefaultServicerStakeWeightCeiling   int64 = 60000000000
```

Comment:
Shouldn't the default values be equivalent to whatever makes the application behave like it does today?

Reply:
> There are very little syntax and structural changes needed for this PR, however there's one mechanism missing from the PR and it's understandable as we've never had to add a parameter before: we need code to process to actually write the params to the state. Luckily we were working in a PR as well to add another parameter, so we will be providing a reference implementation for this later today or tomorrow at the latest.

Cool, hit me up when you have an example/ mechanism for this :)

Reply:
Addressed in [d540a6f](https://github.com/pokt-network/pocket-core/pull/1449/commits/d540a6fdb0bdd6a472d096ac425636f767acf515)